test/math.ts

```diff
@@ -1,69 +1,67 @@
 import * as test from "tape";
-import {evaluate,verify,dedent,testSingleExpressionByList} from "./shared_functions";
+import {evaluate, verify, dedent, testSingleExpressionByList, valueTest} from "./shared_functions";
 
 // Test Constants
-// Would this fail on different architectures than mine? Not according to dcmumentation Maybe we Should
+// Would this fail on different architectures than mine? Not according to documentation Maybe we Should
 // flush out that implicit assumption.
 
-let constants_list : any = [
-  {"Expression":"pi[]",      "Value":"3.141592653589793"},
-  {"Expression":"e[]",       "Value":"2.718281828459045"},
-  {"Expression":"ln2[]",     "Value":"0.6931471805599453"},
-  {"Expression":"log2e[]",   "Value":"1.4426950408889634"},
-  {"Expression":"log10e[]",  "Value":"0.4342944819032518"},
-  {"Expression":"sqrt1/2[]", "Value":"0.7071067811865476"},
-  {"Expression":"sqrt2[]",   "Value":"1.4142135623730951"}
+let delta = 0.00000000001;
```

Comment:
thats really the only question about the epsilon(delta), ieee 32 gets you kind of 6-9 and ieee 64 gets you 15-17. just to complicate things intel machines use 80 internally. and truncate to 64 when you are actually saving to a gpr or memory...so...11 is pretty reasonable to assume 64 bit, but if you want to not assume that (and think js asserts that it must be 64), something like 5 would be more appropriate.

this is fine